src/pages/Models/Details/_model-details.scss

```diff
@@ -4,34 +4,31 @@
 .model-details {
   display: grid;
   grid-template-columns: 230px 1fr;
+  margin-bottom: 2rem;
   padding-bottom: 14rem;
 
-  @media (min-width: $breakpoint-m-large) {
```

Comment:
The stylelint kind of went crazy with this file. It might be best to view the commits for the changes directly. They are limited to some background colours and a hover.